piq/gmsd.py

```diff
@@ -179,70 +281,17 @@ def __init__(self, reduction: str = 'mean', data_range: Union[int, float] = 1.,
         self.beta3 = beta3
             
     def forward(self, prediction: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
-        r"""Computation of Multi scale GMSD as a loss function.
+        r"""Computation of Multi Scale GMSD index as a loss function.
 
         Args:
-            prediction: Tensor of prediction of the network.
-            target: Reference tensor.
+            prediction: Tensor of prediction of the network. Required to be
+                2D (H, W), 3D (C,H,W) or 4D (N,C,H,W), channels first.
+            target: Reference tensor. Required to be 2D (H, W), 3D (C,H,W) or 4D (N,C,H,W), channels first.
 
         Returns:
-            Value of GMSD loss to be minimized. 0 <= GMSD loss <= 1.
+            Value of SSIM loss to be minimized, i.e 1 - `ssim`. 0 <= SSIM loss <= 1. In case of 5D input tensors,
```

Comment:
Do we really support complex tensors? 
```suggestion
            Value of GMSD loss to be minimized. 0 <= GSMD loss <= 1. In case of 5D input tensors,
```

Reply:
Nope